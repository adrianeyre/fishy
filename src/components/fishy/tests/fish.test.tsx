import React from 'react';
import { shallow } from 'enzyme';

import Fishy from '../fishy';
import IFishyProps from '../interfaces/fishy-props';

describe('Fishy', () => {
	it('Should render correctly', () => {
		const defaultProps: IFishyProps = {};
		const fishy = shallow(<Fishy {...defaultProps} />);
		expect(fishy).toMatchSnapshot();
	});
});