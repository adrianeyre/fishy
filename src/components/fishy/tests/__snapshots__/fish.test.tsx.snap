// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fishy Should render correctly 1`] = `ShallowWrapper {}`;
